src/Microsoft.DotNet.Darc/Darc/Operations/VirtualMonoRepo/CodeFlowOperation.cs

```diff
@@ -173,4 +184,86 @@ protected async Task<string> GetSourceMappingNameAsync(NativePath repoPath)
 
         return versionDetails.Source.Mapping;
     }
+
+    /// <summary>
+    /// Fetch subscription metadata and populate command options based on subscription settings.
+    /// This allows the subscription to be simulated using the existing codeflow logic.
+    /// </summary>
+    private async Task<IReadOnlyList<string>> PopulateOptionsFromSubscriptionAsync()
```

Comment:
I don't like that nothing about this method name tells you it's gonna return a list of excluded assets. Think it'd be better if it was a parameter, we passed a list, and this method would fill it